Project/frontend/src/components/Graph/index_visjs.tsx

```diff
@@ -3,6 +3,7 @@ import { Network } from 'vis-network/standalone/esm/vis-network';
 import { useParams } from 'react-router-dom';
 import './index.css';
 import { VISUALIZE_API_PATH } from '../../constant';
+import { FaSearch } from 'react-icons/fa';
```

Comment:
FYI MUI library also has a package for icons  https://mui.com/material-ui/icons/ . Would be great if we use the same library for it well for maintenance purposes.
 273,  Would be nice if you use MUI input component for UX consistency https://mui.com/material-ui/react-text-field/
 281
Same here. Would be nice if you use MUI input component for UX consistency https://mui.com/material-ui/react-text-field/